src/mlir/dialect/VeronaTypes.h

```diff
@@ -55,6 +59,9 @@ namespace mlir::verona
     llvm::ArrayRef<mlir::Type> getElements() const;
   };
 
+  /**
+   * Meet types are unions between types (A | B).
```

Comment:
Copy-and-paste error and the same question regarding defining a new word.

Reply:
oops